string path_to_template

---

string path_to_image
